roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```
@@ -104,10 +104,11 @@
 
 - name: Stop if ip var does not match local ips
   assert:
-    that: ip in ansible_all_ipv4_addresses
+    that: (ip in ansible_all_ipv4_addresses) or (ip in ansible_all_ipv6_addresses)
     msg: "'{{ ansible_all_ipv4_addresses }}' do not contain '{{ ip }}'"
```

better to change this `msg` for IPv6 also to understand what is wrong easily for users.
```suggestion
    msg: "IPv4: '{{ ansible_all_ipv4_addresses }}' and IPv6: '{{ ansible_all_ipv6_addresses }}' do not contain '{{ ip }}'"
```